src/SFA.DAS.ApprenticeCommitments/Application/Commands/CreateApprenticeship/UpdateApprenticeshipCommand.cs

```diff
@@ -0,0 +1,83 @@
+using MediatR;
+using SFA.DAS.ApprenticeCommitments.Apis.InnerApi;
+using SFA.DAS.ApprenticeCommitments.Apis.TrainingProviderApi;
+using SFA.DAS.ApprenticeCommitments.Application.Services;
+using SFA.DAS.ApprenticeCommitments.Application.Services.ApprenticeLogin;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using CommitmentsApprenticeshipResponse = SFA.DAS.ApprenticeCommitments.Apis.CommitmentsV2InnerApi.ApprenticeshipResponse;
+
+#nullable enable
+
+namespace SFA.DAS.ApprenticeCommitments.Application.Commands.CreateApprenticeship
```

Comment:
sorry missed this earlier, the file is in the wrong folder. It should be in UpdateApprenticeship